velox/functions/sparksql/StringToMap.h

```diff
@@ -23,80 +23,111 @@ namespace facebook::velox::functions::sparksql {
 template <typename T>
 struct StringToMapFunction {
   VELOX_DEFINE_FUNCTION_TYPES(T);
-
   // Results refer to strings in the first argument.
   static constexpr int32_t reuse_strings_from_arg = 0;
-
   void call(
       out_type<Map<Varchar, Varchar>>& out,
       const arg_type<Varchar>& input,
       const arg_type<Varchar>& entryDelimiter,
       const arg_type<Varchar>& keyValueDelimiter) {
-    VELOX_USER_CHECK_EQ(
-        entryDelimiter.size(), 1, "entryDelimiter's size should be 1.");
-    VELOX_USER_CHECK_EQ(
-        keyValueDelimiter.size(), 1, "keyValueDelimiter's size should be 1.");
-
-    callImpl(
-        out,
-        toStringView(input),
-        toStringView(entryDelimiter),
-        toStringView(keyValueDelimiter));
+    VELOX_USER_CHECK_GE(
+        entryDelimiter.size(), 1, "entryDelimiter's size should >= 1.");
```

Comment:
It looks like a check for empty string. Error message suggestion: 
> The entry delimiter should not be empty.

Reply:
done